### ISSUE TITLE
Long usage error "You must read the column first"

### ISSUE THREAD

**Issue by Mr-Cheung094** [ChuckBell/MySQL_Connector_Arduino#206]
![image](https://user-images.githubusercontent.com/76932998/220583010-dada974c-5a44-4013-8ae0-01337d5cba38.png)

hello ，Every time you use it for a long time you will always report the error shown on the diagram，I looked at the header files contained in our database and did not find such an error message。Now, I have no way to solve the problem. I hope you can help me。

**Comment by ChuckBell** [comment]
Hi. This means there is more to be read from the server. Be sure you have read the columns and all of the data in your loop per the examples.

**Comment by Mr-Cheung094** [comment]
Yes,I sure. because The database data can be read normally after a short time。Long time use, to repair the charge reset, can be used

**Comment by Stergios96T** [comment]
Hello! I am facing the same error after a while that arduino running
Any solution would be much appreciated!
